index.html

```diff
@@ -1,10 +1,132 @@
+                  <ul>
+                      <li>
+                          <p>Рассмотрим пять причин,по которым Седона круче,чем гранд каньон!</p>
+              </article>
+              <h2>Преимущества</h2>
+              <h3>Настоящий городок</h3>
+             <p>№1</p>
+             <p>Седона-не аттракцион для туристов,там течет своя жизнь</p>
+              <img src"city" width="800" height="256"
+                 alt="Настоящий городок"
+                   <h4>Жилье</
+                       <p>Рекомендуем пожить в настоящем мотеле,все как в кино</p>
+              </li>
+             <li class="food"
+                 " <h4>Еда</h4>
+              <p>Всегда заказывайте фирменный бургер,вы не разочаруетесь</p>
+              </li>
+              <link class="gift"
+                    "<h4>Сувениры</h4>
+              <p>Не только китайского,но и местного производства!</p>
+              </li>
+
+              </ul>
+              </li>
+              <ul>
+          </section>
+          <section>
+              <li class="advanteges">
+                  alt="Мост дьявола" height="256"
+                  <img src="img/bridge" width="800"
+                  <h3>Там есть <br>Мост дьявола</h3>
+                  <p>№2</p>
+                  <p>Да,по нему можно пройти!Если конечно вы осмелитесь</p>
+              </li>
+              </ul>
+              <ul>
+                  <li class="advanteges">
+                      <h3>Небольшая площадь</h3>
+                      <p>№3</p>
+                      <p>Все достопримечателности находятся очень близко</p>
+                  </li>
+                  <li class="advanteges">
+                      <h3>Красивая дорога</h3>
+                      <p>№4</p>
+                      <p>Ехать в Седону из Лас-Вегаса совсем не скучно!</p>
+                  <li class="advanteges">
+                      <h3>Мало туристов</h3>
+                      <p>№5</p>
+                      <p>Большинство едет в Гранд Каньон и толпится там</p>
+                  </li>
+              </ul>
+          </section>
+          <section>
+              <h2>Поиск гостиниц в Седоне</h2>
+              <p>Заинтересовались?</p>
+              <p>Укажите предполагаемые даты поездки,<br />и мы покажем вам лучшие предложения гостиниц в Седоне</p>
+          
+          <div class="form"
+          <a href="#" class="hotels-buttom"
+             <h2 class="form-header">Поиск гостиниц в Седоне</h2>
+</a>
+          </div>
+              </section>
+          <section class="form">
+              <p>Дата заезда</p>
+              <button type="button" class="calendar-form"
+                      <span>Календарь</span>
+              <src height="22" width="21"></src>
+              </button>
+              <p>Дата заезда</p>
+              <button type="button" class="calendar-form"
+                      <span>Дата выезда</span>
+              <src height="22" widht="21"></src>
+                   </button>
```

Comment:
По заданию еще рано делать формы, давай пока ограничимся кнопкой :)
```suggestion
              <button type="button">Поиск гостиницы в Седоне</button>
```
А вообще формы делаются через специальный тег `form`